ts/components/leftpane/ActionsPanel.tsx

```diff
@@ -238,6 +240,20 @@ export const ActionsPanel = () => {
     return () => clearTimeout(timeout);
   }, []);
 
+  const globalUnreadMessageCount = useSelector(getGlobalUnreadMessageCount);
+  const unreadToShow = globalUnreadMessageCount;
+
+  // Reuse the unreadToShow from the global state to update the badge count
+  useThrottleFn(
+    () => {
+      if (unreadToShow !== undefined) {
+        ipcRenderer.send('update-badge-count', unreadToShow);
+      }
+    },
+    2000,
+    []
+  );
```

Comment:
have you checked the docs? https://github.com/streamich/react-use/blob/master/docs/useThrottleFn.md

Reply:
yes i have and it seems they're out of date, because I wasn't able to add a dependency array at the end, the way the docs were suggesting. It would only accept an empty array.